qiskit/aqua/operators/__init__.py

```diff
@@ -153,7 +153,10 @@
                          Suzuki, QDrift)
 
 # Convenience immutable instances
-from .operator_globals import EVAL_SIG_DIGITS, X, Y, Z, I, CX, S, H, T, Swap, Zero, One, Plus, Minus
+from .operator_globals import (EVAL_SIG_DIGITS,
```

Comment:
Can you add 'CZ' to the global operators docstring here above that lists these out for the docs

Reply:
Done!